### PR TITLE
add quoteType to get-quote-v2

### PR DESCRIPTION
**Pull request by jerryafr**

### CODE REVIEW

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 1]
```diff
@@ -274,6 +279,7 @@ Learn more about Chainflip's [$USDC Denominated Pools](../../../concepts/swaps-a
     ],
     "estimatedDurationSeconds": 1830,
     "estimatedPrice": "0.05",
+    "QuoteType": "REGULAR",
```
Comment (zoheb391) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:282]: wrong case

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 2]
```diff
@@ -306,6 +312,7 @@ Learn more about Chainflip's [$USDC Denominated Pools](../../../concepts/swaps-a
           "fee": { "chain": "Ethereum", "asset": "USDC", "amount": "18062" }
         }
       ],
+      "QuoteType": "REGULAR",
```
Comment (zoheb391) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:315]: and here

**Comment by CumpsD** [comment]
This rocks! quoteType is much clearer than having to find a dcaParams property to determine the type :)

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx [hunk 3]
```diff
@@ -204,6 +207,7 @@ Learn more about Chainflip's [$USDC Denominated Pools](../../../concepts/swaps-a
       "chunkIntervalBlocks": 2,
       "numberOfChunks": 1
     },
+    "QuoteType": "DCA",
```
Comment (CumpsD) [inline pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:210]: and here ;)

File: pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx (hunk 1 above)
Comment (CumpsD) [reply pages/swapping/integrations/javascript-sdk/get-quote-v2.mdx:282]: also the schema above says `type` not `quoteType`, so not just case but also naming `type: QuoteType;`

**Pull request merged**